README.md

```diff
@@ -50,10 +50,20 @@ sh -c "$(curl -fsSL https://raw.githubusercontent.com/streamnative/pulsarctl/mas
 
 #### Install pulsarctl binary on Windows
 
+If you have command-line shell tools, such as PowerShell. You can download pulsarctl by this way:
+
+```bash
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/streamnative/pulsarctl/master/install.sh)"
+```
+
+Or you can download the executable directly.
+
 ``` bash
 curl -LO https://github.com/streamnative/pulsarctl/releases/download/v0.1.0/pulsarctl.exe
```

Comment:
don't you need to fix this? we don't ship `pulsarctl.exe`.

Reply:
Ah. Sorry about that. I need. Will fix it.